js/king.js

```diff
@@ -0,0 +1,53 @@
+import {Piece} from "./piece.js";
+
+export class King extends Piece {
+    /**
+     * Constructor: inherits from abstract Piece class
+     * @param id piece id
+     * @param rowPos current row position
+     * @param colPos current column position
+     * @param playerId player id
+     */
+    constructor(id, rowPos, colPos, playerId) {
+        super();
+        this.id = id;
+        this.rowPos = rowPos;
+        this.colPos = colPos;
+        this.playerId = playerId;
+        this.unicodeChar = "&#9812;";
+    }
+
+    /**
+     * Gets a list of possible moves the rook piece can make including attack moves
```

Comment:
Docstring on line `21` should refer to the king piece instead of the rook piece.